package.json

```diff
@@ -25,15 +25,20 @@
     "test:coverage": "jest --coverage"
   },
   "dependencies": {
+    "react-scripts": "^5.0.1",
     "requireindex": "^1.2.0"
   },
   "peerDependencies": {
     "eslint": ">=4.0.0"
   },
   "devDependencies": {
-    "eslint": "^6.8.0",
+    "eslint": "^8.13.0",
     "eslint-config-godaddy": "^6.0.0",
     "eslint-plugin-jest": "^26.1.1",
+    "eslint-plugin-mocha": "^10.0.4",
     "jest": "^24.8.0"
+  },
+  "jest": {
+    "testEnviroment": "node"
```

Comment:
This should not be necessary as `testEnvironment` defaults to `"node"` ([docs](https://jestjs.io/docs/configuration#testenvironment-string)).